src/components/Home.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from 'react'
+import { ArticleCard } from './ArticleCard'
+
+export const Home = () => {
+
+const [articles,setArticles]=useState([])
+
+    useEffect(() => {
+        fetch(`https://news-website-0p9e.onrender.com/api/articles`).then((response) => {
+        return response.json()
+        }).then((response) => {
+            console.log(response.article)
+        setArticles(response.article)
```

Comment:
You don't seem to be using an `isLoading` pattern - this means the section holding all the articles is just blank while it fetches the data, instead of saying something like 'Loading...'.

Please implement the `isLoading` pattern to improve the user experience.